platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx

```diff
@@ -1,84 +1,136 @@
+
+   - `TOWER_DATA_STUDIO_ALLOWED_WORKSPACES`: Enable [Data Studios](https://docs.seqera.io/platform/latest/data/data-studios) in the specified workspaces.
+   - `TOWER_DATA_EXPLORER_ENABLED`: Specifies whether to enable Data Explorer. You must enable Data Explorer to mount data inside a data studio instance.
+   - `TOWER_DATA_STUDIO_CONNECT_URL`: Specifies the URL of the Data Studios connect proxy, such as `https://connect.tower.example.com/`.
+   - `TOWER_OIDC_REGISTRATION_INITIAL_ACCESS_TOKEN`: An access token that is used for registering new clients with Seqera Platform. Any alphanumeric value is allowed.
+   - `TOWER_OIDC_PEM_PATH`: The file path to a PEM certificate that is used for signing the OIDC tokens for the OpenID connect provider.
+
+1. Open the `docker-compose.yml`, and uncomment the volume mount for the PEM key file for the `backend` service in the `volumes` list. Your PEM file must be named `data-studios-rsa.pem`.
+
+    ```yaml
+        volumes:
+          - $PWD/tower.yml:/tower.yml
+          # Data studios RSA key is required for the data studios functionality. Uncomment the line below to mount the key.
+          #- $PWD/data-studios-rsa.pem:/data-studios-rsa.pem
+    ```
+
+1. Start your Platform instance: `docker compose -d up`.
+
+## Kubernetes
+
+This procedure describes how to configure Data Studios for Kubernetes deployments of Seqera Platform.
+
+### Procedure
+
+1. Download the Kubernetes manifests for the Data Studios service:
+
+    - [Ingress](./_templates/k8s/data_studios/ingress.aks.yml)
+    - [Proxy config](./_templates/k8s/data_studios/proxy-config.yml)
+    - [Proxy](./_templates/k8s/data_studios/proxy.yml)
+    - [Server](./_templates/k8s/data_studios/server.yml)
+
+1. Edit the `server.yml` file and set the `REDIS_ADDRESS` environment variable to the host name or IP address of the Redis server that you configured for Platform.
+
+1. Edit the `proxy.yml` file and set the following variables:
+
+    - `REDIS_SERVER`: Specify the host name or IP address of the Redis server that you configure for Platform.
+    - `TOWER_CONNECT_DOMAIN`: Specify a base domain name for the connect proxy subdomain. For example, if you specify `example.com` then `connect.example.com` is the fully qualified domain name for the proxy.
+    - `TOWER_BASE_URL`: Specifies the base URL for your Platform installation, such as `https://tower.example.com/`.
+
+1. Change your Kubernetes context to the one where your Platform instance runs:
+
+    ```
+    kubectl config set-context --current --namespace=<namespace>
+    ```
+
+1. Create a Secret that contains the initial access token. Any alphanumeric value is allowed.
+
+    ```yaml
+    apiVersion: v1
+    kind: Secret
+    type: Opaque
+    metadata:
+      name: connect-proxy-secret
+      namespace: connect-stage
+    stringData:
+      INITIAL_ACCESS_TOKEN: <INITIAL_ACCESS_TOKEN>
```

Comment:
Changed variable name from to INITIAL_ACCESS_TOKEN to OIDC_CLIENT_REGISTRATION_TOKEN, for better understanding